telegram/ext/filters.py

```diff
@@ -1966,6 +1967,37 @@ def filter(self, message: Message) -> bool:
     .. versionadded:: 20.0
     """
 
+    class _CustomEmoji(MessageFilter):
+        __slots__ = ()
+
+        def filter(self, message: Message) -> bool:
+
+            return bool(message.sticker) and (
+                message.sticker.type == StickerTypeEnum.CUSTOM_EMOJI  # type: ignore
+            )
+
+    CUSTOM_EMOJI = _CustomEmoji(name="filters.Sticker.CUSTOM_EMOJI")
+    """Messages that contain :attr:`telegram.Message.sticker` and are a custom emoji, so the
+    :paramref:`type <telegram.Sticker.type>` equals :tg-const:`telegram.Sticker.CUSTOM_EMOJI`.
+
+    .. versionadded:: 20.0
+    """
+
+    class _Mask(MessageFilter):
+        __slots__ = ()
+
+        def filter(self, message: Message) -> bool:
+            return bool(message.sticker) and (
+                message.sticker.type == StickerTypeEnum.MASK  # type: ignore
+            )
+
+    MASK = _Mask(name="filters.Sticker.MASK")
+    """Messages that contain :attr:`telegram.Message.sticker` and are a mask, so the
+    :paramref:`type <telegram.Sticker.type>` equals :tg-const:`telegram.Sticker.MASK`.
+
+    .. versionadded:: 20.0
+    """
+
```

Comment:
Are these actually possible? If I send a message with a single custom emoji, will I have `message.sticker` or will I have `message.text` + `message.entities`? If I send a photo with a mask on it, will I have `message.sticker` or will I have `message.photo`?

Reply:
just a small test for ptbgithubbridge - ignore me

Reply:
another test